core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -83,6 +89,11 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
   def main(args: Array[String]) {
 
     info("Starting mirror maker")
+    println("WARNING: The default partition assignment strategy of mirror maker will change from 'range' to 'roundrobin' " +
```

Comment:
@vahidhashemian MM runs as a distributed system. So to upgrade configuration without downtime, you need to do a rolling restart of the MM processes. This means that some instances in the cluster will be on the old config and some will be on the new config. We could tell users that they have to shutdown all of their MM processes, perform the upgrade, and then restart, but that's not too great, right?